### PR TITLE
[stable/rabbitmq-ha] Only allow guest if it was enabled

### PR DESCRIPTION
**Pull request by zerkms**
#### What this PR does / why we need it:

at the moment the guest user is exposed even if the cluster user was renamed using `rabbitmqUsername` value .

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/rabbitmq-ha/templates/configmap.yaml [hunk 1]
```diff
@@ -73,9 +73,11 @@ data:
     cluster_formation.node_cleanup.only_log_warning = true
     cluster_partition_handling = {{ .Values.rabbitmqClusterPartitionHandling }}
 
+    {{- if eq .Values.rabbitmqUsername "guest" }}
```
Comment (steven-sheehy) [inline stable/rabbitmq-ha/templates/configmap.yaml:76]: It's unfortunate the chart's default user is guest, we really should change that at some point.

File: stable/rabbitmq-ha/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name: rabbitmq-ha
 apiVersion: v1
 appVersion: 3.8.0
-version: 1.36.0
+version: 1.36.1
```
Comment (steven-sheehy) [inline stable/rabbitmq-ha/Chart.yaml]: Need a bump as another PR got in first.

Comment (zerkms) [reply stable/rabbitmq-ha/Chart.yaml]: @steven-sheehy yep, I noticed that and wasn't sure if either or both would be accepted, then the other would conflict and I'll resolve right after that.

**Comment by steven-sheehy** [comment]
/lgtm
Thanks!

**Pull request merged**